themes/qgrid/source/js/menu.js

```diff
@@ -15,26 +15,196 @@ function toggleVisibility(e) {
 	}
 }
 
+function replaceSymbol(text) {
```

Comment:
please make a better name why it's called `replaceSymbol`? from outside user it doesn't make any sense
`replaceWhiteSpace` is better for instance